klt-cert-manager/config/default/kustomization.yaml

```diff
@@ -12,7 +12,7 @@ namePrefix: klt-
 #commonLabels:
 #  someName: someValue
 
-bases:
+resources:
```

Comment:
why did you change that? also if you want to change it, please change it in all kustomization files

Reply:
it screamed at me locally that it has been deprecated 😅 
I will research a bit more and provide a follow-up for this

Reply:
AFAIK it's deprecated but left in because `resources` can have different behaviour in some cases and there is no 1:1 replacement for `bases`